backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/error/AdIssueFactory.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2022 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.error;
+
+import de.metas.util.Check;
+import de.metas.util.Services;
+import de.metas.util.StringUtils;
+import lombok.experimental.UtilityClass;
+import org.adempiere.ad.service.ISystemBL;
+import org.adempiere.util.net.NetUtils;
+import org.compiere.Adempiere;
+import org.compiere.model.I_AD_Issue;
+import org.compiere.model.I_AD_System;
+import org.compiere.util.DB;
+
+import java.util.Properties;
+
+import static org.adempiere.model.InterfaceWrapperHelper.newInstance;
+
+@UtilityClass
+public class AdIssueFactory
+{
+	public I_AD_Issue createNewIssueRecord(final Properties ctx)
```

Comment:
IMHO we can get rid of `ctx` param

Reply:
and btw, i would call the method `prepareNewIssueRecord` because it's not actually creating a new issue (yet)